lib/datadog/core/configuration/settings.rb

```diff
@@ -438,7 +445,7 @@ def initialize(*_)
           # @default `DD_RUNTIME_METRICS_ENABLED` environment variable, otherwise `false`
           # @return [Boolean]
           option :enabled do |o|
-            o.env Core::Runtime::Ext::Metrics::ENV_ENABLED
+            o.env [Core::Runtime::Ext::Metrics::ENV_ENABLED]
```

Comment:
Nit non-blocking: are `[]` necessary here?

Reply:
I was experimenting with the Array syntax and forgot to remove it from this line 😓